src/dataaccess/GenericSmartOnFhirDstu2DataSource.jsx

```diff
@@ -0,0 +1,17 @@
+import McodeV05SmartOnFhirDataSource from "./McodeV05SmartOnFhirDataSource";
+import processFHIRResources from './utils/fhir-entry-processor';
+import mappers from 'fhir-mapper';
+
+class GenericSmartOnFhirDstu2DataSource extends McodeV05SmartOnFhirDataSource {
+    constructor(props) {
+        super();
+        this.mapper = props && props.mapper ? mappers[props.mapper] : null;
```

Comment:
For posterity: `props.mapper` is a string so this can only reference mappings that are already defined in the fhir-mapper library. I was about to propose making this more generic to support defining arbitrary mappings in the props, but then I realized if we have a need to create a specific mapping, then we could just define the mapping in a new DataSource, ex "SpecificEhrVendorSmartOnFhirDataSource", so I'm fine with how this works now.